src/main/__tests__/tab-listeners.test.js

```diff
@@ -71,17 +63,20 @@ describe('Main :: Tab listeners test suite', () => {
     test('Previous saved tabs in loaded settings, should add tabs to manager and activate them as they are added', () => {
```

Comment:
Update the description so that it matches what's happening.

(I understand that in the settings module, the ensureDefaultValues is setting the active tab to the first tab in case the activeTab ref doesn't exist.)